website/content/platform/development/contributing/how-to/add_obbject_extension.md

```diff
@@ -1,24 +1,30 @@
 ---
-title: OBBject Extensions
+title: How To Add OBBject Extensions
 sidebar_position: 8
 description: This page provides information about how to write extensions for the OpenBB OBBject class.
 keywords:
 - OBBject
 - Python
 - Development
 - OpenBB Platform
+- extensions
+- obbject extension
+- accessor
+- decorator
+- how-to
+- contributing
 ---
 
 import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
 
-<HeadTitle title="OBBject Extensions - Developer Guidelines - Development | OpenBB Platform Docs" />
+<HeadTitle title="How To Add OBBject Extensions - Developer Guidelines - Development | OpenBB Platform Docs" />
 
-As mentioned, OpenBB provides some basic methods for interacting with common data structures that will be seen in the results attribute.
-If you are working with custom data, or adding new endpoints, it is possible that you will want to have your own methods for interacting with the data.
-The OpenBB Platform provides a way to do this by extending the OBBject class.
-The architecture for extensions was designed similar to how extensions and accessors are done in pandas, and relies on plugins through the poetry dependency management package.
+OpenBB provides some basic methods for interacting with common data structures that will be seen in the results attribute of the [`OBBject`](platform/development/obbject.md)
+If you are working with custom data, or adding new endpoints, it is possible that you will want to have your own methods for interacting with the data, and the OpenBB Platform provides a way to extend the OBBject class.
 
-This page will go through the steps of developing a simple, custom extension for the OBBject class.
+The architecture for extensions was designed to be similar to extensions and accessors for Pandas, and relies on plugins through the Poetry dependency management package.
```

Comment:
Is this important to know in order to follow the example?

Reply:
Why not? It provides context for the reader who might be familiar with Pandas extensions already.

Reply:
We can keep it, but the chances are low and it might seem as a digression.